VEHICLE/GetConvertibleRoofState.md

```diff
@@ -7,21 +7,33 @@ ns: VEHICLE
 // 0xF8C397922FC03F41 0x1B09714D
 int GET_CONVERTIBLE_ROOF_STATE(Vehicle vehicle);
 ```
+Returns the convertible state of the specified vehicle.
+
+Got a "6" return value but not sure about what the value means
+6 -> unknown (Stopped but not fully open ?)
```

Comment:
if this is actually valid it should be put with return types

Reply:
This was there before I made any changes, wasn't able to reproduce a situation where the roof stopped mid-way.